Tests/SwinjectAutoregistrationTests/OperatorsSpec.swift

```diff
@@ -108,8 +108,8 @@ class OperatorsSpec: QuickSpec {
                 //let service2_name: Service2 = container ~> (Service2.self, name: "MyService2", arguments: "Arg1", 5)
                 
                 //This is deprecated
-                let service3 = container ~> (Service3.self, arguments: "Arg1", 5, 0.2)
-                let service3_name = container ~> (Service3.self, name: "MyService3", arguments: "Arg1", 5, 0.2)
+                let service3 = container ~> (Service3.self, arguments: ("Arg1", 5, 0.2))
+                let service3_name = container ~> (Service3.self, name: "MyService3", arguments: ("Arg1", 5, 0.2))
```

Comment:
This now covers the same cases as previous 2 tests - are we conciously dropping support for deprecated syntax? If so, deprecated function should be removed as well

Reply:
I am up for dropping the deprecated syntax ... it has been several years since we deprecated it.